about.html

```diff
@@ -0,0 +1,174 @@
+                    </p>
+                    <div class="m-5 pb-5">
+                        <p>
+                            Join a local group and meet local developers with similar interests.
+                        </p>
+                        <a class="btn-lg text-dark fw-semibold" href="#" role="button">Find a Google Developer Group near you ></a>
+                    </div>
+                </div>
+            </div>
+            <div class="d-flex justify-content-center border border-2">
+                <div class="text-center">
+                    <div class="main-title position-relative m-5">
+                        <h2 class="fw-semibold">Google Developer Group</h2>
+                    </div>
+                    <p class="d-block d-md-none white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+                        Google Developer Groups represents a network of technical
+                        communities that offer opportunities for professional development
+                        and connection.
+                    </p>
+                    <p class="d-none d-md-block white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+                        Google Developer Groups represents a network of technical
+                        communities that offer <br />opportunities for professional
+                        development and connection. Those who join can learn about the
+                        latest Google<br /> tools and technologies, and can meet with 
+                        local developers with similar interests.
+                    </p>
+                    <div class="m-5">
+                        <img class="w-25 border border-2" src="img/gdg-member-badge.png" alt=""/>
+                    </div>
+                </div>
+            </div>
+            <div class="d-flex justify-content-center">
+                <div class="text-center">
+                    <div class="main-title position-relative m-5">
+                        <h2 class="fw-semibold">Earnable badges</h2>
+                    </div>
+                    <p class="white-bg mb-5 pb-4 ps-5 pe-5 text-wrap">
+                        Earn these badges when you visit and register for 2022 DevFest
+                        event.
+                    </p>
+                    <div class="d-flex d-md-none m-5 flex-column justify-content-center align-items-center">
+                        <img class="w-50 m-1 border border-2" src="img/badge-visit.svg" alt=""/>
+                        <img class="w-50 m-1 border border-2" src="img/badge-register.svg" alt=""/>
+                    </div>
+                    <div class="d-none d-md-flex m-5 justify-content-center">
+                        <img class="w-50 m-1 border border-2" src="img/badge-visit.svg" alt=""/>
+                        <img class="w-50 m-1 border border-2" src="img/badge-register.svg" alt=""/>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
**I had to highlight this section as the partners section seems to not be included in your code base**

- [ ] In the mobile version right after the "Earnable badges" you are required by the provided design to have a "Partners section" which must have a black background as well. KIndly implement this. 

![image](https://user-images.githubusercontent.com/86318284/229050286-2b856a87-22d3-4923-a7e8-99af6917454a.png)

**Remember:** this section is hidden on desktop view :slightly_smiling_face:

Reply:
@Nemwel-Boniface I could not find where this requirement was required. However, I implemented your suggestion.

Reply:
Well resolved :heavy_check_mark: